packages/node-core/src/utils/graphql.ts

```diff
@@ -13,10 +13,14 @@ import {
   isNull,
   GraphQLEntityField,
   GraphQLJsonFieldType,
+  blake2AsHex,
 } from '@subql/utils';
 import {ModelAttributes, ModelAttributeColumnOptions} from '@subql/x-sequelize';
 import {isArray, isObject} from 'lodash';
-import {enumNameToHash} from '../db';
+
+export function enumNameToHash(enumName: string): string {
+  return blake2AsHex(enumName).substr(2, 10);
```

Comment:
String.substr has been officially marked as not deprecated.

Reply:
I've just moved this function to fix an issue with circular imports, I don't want to be modifying those things as it's out of scope